site/assets/scss/_callouts.scss

```diff
@@ -27,7 +28,8 @@
 // Variations
 @each $variant in $bd-callout-variants {
   .bd-callout-#{$variant} {
-    --bd-callout-bg: var(--bs-gray-200); // Boosted mod
-    --bd-callout-border: var(--bs-#{$variant}); // Boosted mod
+    --bd-callout-color: var(--bs-emphasis-color); // Boosted mod
+    // Boosted mod: no `--bs-callout-bg`
+    --bd-callout-border: var(--bs-#{$variant}-border-subtle);
```

Comment:
Didn't we said that we wanted to discourage subtle variables for now ? (since it is the doc, we might let it as-is)

Reply:
Its on our side but might be fine as-is.